db/version_set_test.cc

```diff
@@ -2862,11 +2862,12 @@ class VersionSetTestEmptyDb
     assert(nullptr != log_writer);
     VersionEdit new_db;
     if (db_options_.write_dbid_to_manifest) {
+      ASSERT_OK(SetIdentityFile(env_, dbname_));
```

Comment:
This test suite never created identity file when calling this method.

Reply:
Not really sure what this PR comment indicates.

Reply:
Sorry it was not clear before. I meant to explain why I have to add this line of code. The reason is that the call to `impl->GetDbIdentityFromIdentityFile(&db_id)` below was not returning ok status and that's because there is no IDENTITY file in the specified db path (`dbname_`).  The reason for no IDENTITY file is that no DB was actually opened/created.